controllers/organization_members_controller.go

```diff
@@ -0,0 +1,100 @@
+package controllers
+
+import (
+	"context"
+
+	"go.uber.org/multierr"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	controlv1 "github.com/appuio/control-api/apis/v1"
+)
+
+// OrganizationMembersReconciler reconciles OrganizationMembers resources
+type OrganizationMembersReconciler struct {
+	client.Client
+	Recorder record.EventRecorder
+	Scheme   *runtime.Scheme
+
+	// UserPrefix is the prefix applied to the user in the RoleBinding.subjects.name.
+	UserPrefix  string
+	MemberRoles []string
+}
+
+//+kubebuilder:rbac:groups=appuio.io,resources=organizationmembers,verbs=get;list;watch;update;patch
+//+kubebuilder:rbac:groups=appuio.io,resources=organizationmembers/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=rolebindings,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="",resources=events,verbs=create;patch
+
+// Needed so that we are allowed to delegate common member roles
+//+kubebuilder:rbac:groups="rbac.appuio.io",resources=organizations,verbs=get;list;watch;create;delete;patch;update
+//+kubebuilder:rbac:groups="organization.appuio.io",resources=organizations,verbs=get;list;watch;create;delete;patch;update
+//+kubebuilder:rbac:groups="appuio.io",resources=teams,verbs=get;list;watch;create;delete;patch;update
```

Comment:
I don't think this is needed? What do we do with members though that are only in teams? Do they get to edit their organization?

Reply:
Team membership is not handled in any way for this PR. I don't think we ever talked about that. I think that's out of scope.

We need these permissions as the default `org-admin` role gives these permissions, so the controller needs them too. Otherwise k8s will complain when it tries to create the rolebinding

Reply:
Ahh yeah i already forgot again how giving permissions worked 🤦‍♀️